tests/mamba/selective_state_update_triton.py

```
@@ -142,70 +173,126 @@ def _selective_scan_update_kernel(
+                    offs_m[:, None] * dstate + offs_n[None, :]
+                )
+                tl.store(cache_ptrs, state.to(cache_ptrs.dtype.element_ty), mask=mask)
+
+        out = tl.sum(state * C[None, :], axis=1)
+        if HAS_D:
+            out += x * D
+        if HAS_Z:
+            out *= z * tl.sigmoid(z)
+        tl.store(out_ptrs, out, mask=offs_m < dim)
+
+        current_step_idx += 1  # noqa: SIM113
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `SIM113`)

Remove unused `noqa` directive

(RUF100)